tests/clone.rs

```diff
@@ -6,9 +6,12 @@ use derive_restricted::derive_where;
 // #[derive_where(T: Clone; Clone)]
```

Comment:
```suggestion
// #[derive_where(T; Clone)]
```
Maybe better make this an actual test without the generic. As Generics on Units are not supported by Rust.

Reply:
I think we can remove the whole thing. The crate shouldn't handle things without generics anyway, because the standard library does that already perfectly.

Reply:
Yeah, we should just add a reasonable error than that tells the user to use `#[derive]`

Reply:
I noted this in #2.